### ISSUE TITLE
UnicodeModelsTest can safely be removed

### ISSUE THREAD

**Issue by benjaoming** [learningequality/ka-lite#3423]
I'm suggesting to completely remove this test because it's stupid:

1. Testing tests!? Test inception. Mind blown.
1. Coverage is done with a coverage tool
1. The test doesn't do what it says, it just tests that a new model hasn't been added without being mentioned in the test.. it's not actually testing if the __unicode__ method has a test or not.
1. Also, it didn't even work before, so who really cares? :)

So after I found that the path "main.models" and "updates.models" needed to be prefixed "kalite.main.models" and "kalite.updates.models" in their respective `unicode_tests` modules, I found that they were producing this error (Python 2.7 only, because that's the only Python where they work with):

    ======================================================================
    FAIL: test_unicode_class_coverage (kalite.main.tests.unicode_tests.MainUnicodeModelsTest)
    ----------------------------------------------------------------------
    Traceback (most recent call last):
    File "/home/travis/build/benjaoming/ka-lite/kalite/main/tests/unicode_tests.py", line 23, in test_unicode_class_coverage
    known_classes = [ExerciseLog, UserLog, UserLogSummary, VideoLog],
    File "./python-packages/fle_utils/testing/unicode.py", line 14, in check_unicode_class_coverage
    self.assertTrue(not set(found_classes) - set(known_classes), "The following models have __unicode__ methods that need to have tests written: %s" % ", ".join([str(cls) for cls in set(found_classes) - set(known_classes)]))
    AssertionError: False is not true : The following models have __unicode__ methods that need to have tests written: <class 'kalite.main.models.ContentLog'>, <class 'kalite.main.models.AttemptLog'>

This is the deceitful code block in fle_utils.testing.unicode:

```python
        # Make sure we're testing all classes
        #   NOTE: we're not testing UserProfile
        found_classes = filter(lambda class_obj: hasattr(class_obj, "__unicode__"), all_classes_in_module(models_module))
        self.assertTrue(not set(found_classes) - set(known_classes), "The following models have __unicode__ methods that need to have tests written: %s" % ", ".join([str(cls) for cls in set(found_classes) - set(known_classes)]))
```

**Comment by MCGallaspy** [comment]
@benjaoming I discovered the same thing while reviewing another PR. I think we can safely remove these tests.

**Comment by benjaoming** [comment]
Cool, thanks @MCGallaspy... I remember you mentioned it quite recently.

The fix is in my [setuptools-installer branch](https://github.com/benjaoming/ka-lite/tree/setuptools-installer) and I won't cherry-pick it just now... I'm hoping the whole thing with all its hazardous new code can be accepted at some point :)

**Comment by aronasorman** [comment]
I don't see any need to cherry-pick this to 0.13, so I'm changing the milestone to 0.14 (aka target develop).

**Comment by benjaoming** [comment]
Yes, no prob, as per last meeting discussion that all of the changes for the setuptools installer are destined for 0.14.

**Comment by benjaoming** [comment]
It's now in #3404 because that's the PR where it was originally blocking...

**Comment by benjaoming** [comment]
Merged in #3404
